### ISSUE TITLE
Offline password bruteforcing via cookies and remember me

### ISSUE THREAD

**Issue by jvoisin** [airsonic/airsonic#937]
The cookie used for the remember me feature (`remember-me`) is storing the user password as md5: should an attacker be able to obtain it, they'll be able to bruteforce the password of the user.

```python
>>> my_cookie = 'anZvaXNpbjoxNTU1MDE4ODMzOTU1OmZkNTU0ZDE4ZjVjNDAxZjIwNGM0ZTRjYWIxZGRhOD'
>>> hashlib.md5('jvoisin:1555018833955:password:airsonic').hexdigest()
'fd554d18f5c401f204c4e4cab1dda88f'
>>> base64.b64encode('jvoisin:1555018833955:fd554d18f5c401f204c4e4cab1dda88f')
'anZvaXNpbjoxNTU1MDE4ODMzOTU1OmZkNTU0ZDE4ZjVjNDAxZjIwNGM0ZTRjYWIxZGRhODhm'
>>> _ == my_cookie
True
>>>
```

A ghetto-mitigation would be to use a super-long-and-random salt for the [`.key` parameter]( https://github.com/airsonic/airsonic/blob/master/airsonic-main/src/main/java/org/airsonic/player/security/GlobalSecurityConfig.java#L165 ) instead of a static one.

**Comment by jvoisin** [comment]
This is `CVE-2019-10907`.

**Comment by jvoisin** [comment]
This issue was fixed in 3e07ea52885f88d3fbec444dfd592f27bfb65647
